wip-pdf.css

```diff
@@ -0,0 +1,29 @@
+@import url('slide-files/fonts/jost/jost.css');
```

Comment:
Should this file be here?

Reply:
Yeah, wip-pdf.css is slated to replace pdf.css once issue #80 is resolved.  I pledge to keep tabs on this.